### PR TITLE
Bug 1264848 - [TV][Home] Implement "remove folder" button in folder card picker

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=142cf635dd907b7f2b77079b0ad2369b68693a26

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fd0b008b1197014aa0aa5bd0c08876c261522d69

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -662,6 +662,10 @@
         this._cardPicker.saveToNewFolder(this.cardScrollable.currentIndex);
       } else if (this._cardPicker.mode === 'update') {
         this._cardPicker.updateFolder();
+        var folder = this._cardPicker.folder;
+        if (folder.isEmpty()) {
+          this.cardManager.removeCard(folder);
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: Please move this manipulation into `cardPicker`.

Supposedly to be in `_saveToFolderHelper`.

File: tv_apps/smart-home/bower_components/smart-button/style.css [hunk 2]
```diff
@@ -492,6 +492,15 @@ smart-button[type="rounded-corner-text"].primary.pressed {
...
 }
 
+smart-button[type="rounded-corner-text"].danger,
+smart-button[type="rounded-corner-text"].danger.focused {
+  background-color: #e63d2f;
+}
+
+smart-button[type="rounded-corner-text"].danger.pressed {
+  background-color: #ac2e23;
+}
```
Comment (rexboy7) [inline tv_apps/smart-home/bower_components/smart-button/style.css]: Please update `smart-button` repository if you want to make change of this.

Or you can just add them in card_picker.css.

File: tv_apps/smart-home/js/home.js (hunk 1 above)
Comment (luke-chang) [inline tv_apps/smart-home/js/home.js]: Good point! I should have thought about this way.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=86bef0b7c8f9e9da59feebcf7c913052d133ee66

**Pull request merged**